script.js

```diff
@@ -1,12 +1,207 @@
+      <div class="col" style="height: 550px">
+        <div class="card h-100 px-0 mx-1">
+          <img src="${
+            show.image !== null ? show.image.medium : ""
+          }" style="height: 250px" class="card-img-top mx-0" alt="...">
+          <div class="card-body mb-4">
+          <h5 class="text-primary">${show.name}</h5>
+          <span class="badge text-bg-dark">Rated: ${show.rating.average}</span>
+          <span class="badge text-bg-dark">Genres: ${show.genres}</span>
+          <span class="badge text-bg-dark">Status: ${show.status}</span>
+          <span class="badge text-bg-dark">Runtime: ${show.runtime}</span>
+          <p>${
+            show.summary.split(" ").length >= 25
+              ? show.summary.split(" ").slice(0, 25).join(" ").concat("...")
+              : show.summary.split(" ").slice(0, 25).join(" ")
+          }</p>
+          </div>
+        </div>
+      </div>
+    `;
+    showEpisodseRow.innerHTML = output;
+  });
+
+  // Create option for each show in select after loading page
+  allShows.forEach((show) => {
+    const option = document.createElement("option");
+    option.value = show.name;
+    option.textContent = show.name;
+    selectShows.appendChild(option);
+  });
+}
+
+// Search specific Shows based on select
+async function selectShow(event) {
+  if (event.target.value == "allSerials") {
+    setup();
+    //Remove options from episodes's select
+    const options = selectEpisodes.getElementsByTagName("option");
+    for (let i = options.length - 1; i > 0; i--) {
+      selectEpisodes.removeChild(options[i]);
+    }
+    searchUser.value = "";
+    searchUser.disabled = true;
+  } else {
+    searchUser.value = "";
+    searchUser.disabled = false;
+    //Remove options from episodes's select
+    const options = selectEpisodes.getElementsByTagName("option");
+    for (let i = options.length - 1; i > 0; i--) {
+      selectEpisodes.removeChild(options[i]);
+    }
+    // Find selected movie
+    const specificShow = allShows.find(
+      (show) => show.name === event.target.value
+    );
+    // Determine show's id
+    const specificShowID = specificShow.id;
+    // Get all show's episodes
+    allEpisodes = await getAllAvailableEpisodes(specificShowID);
+    // make card for each episode
+    makePageForEpisodes(allEpisodes);
+  }
+}
+
+//Get all episodes via fetch
+async function getAllAvailableEpisodes(showId) {
+  // await response of the fetch call
+  const response = await fetch(
+    `https://api.tvmaze.com/shows/${showId}/episodes`
+  );
+  // only proceed once its resolved
+  const data = await response.json();
+  // only proceed once second promise is resolved
+  return data;
+}
+
+//make card for each episode
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  let output = "";
+  episodeList.forEach((episode) => {
+    output += `
+      <div class="col">
```

Comment:
This works well, but doesn't use the DOM APIs - can you give it a try using the DOM APIs to make these elements?